### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Braedon**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict'
-const server = require('server')
-const config = require('server/config.js')
+const server = require('./server')
```
Comment (Nopzen) [inline index.js]: I would pefere not to have these kinda paths it makes it really horrible to go through a path like this:
`require('../../../../../../../../../../something'`) i'am working on a fix for this already witch is basicly just to make the node app read `NODE_PATH=.` i will fix this asap so just revert these back to relative paths :)

File: routes/index.js [hunk 2]
```diff
@@ -4,9 +4,6 @@ const router = express.Router()
 
 const journalist = require('./journalist')
 
-router.route('/').get((req, res) => {
```
Comment (Nopzen) [inline routes/index.js]: Sure, it was just for my own reminder :).

**Comment by Braedon** [comment]
Committing the `.env` was just a hack to get it to run in the docker container. The challenge is that when running with docker there's no need for a `.env` file inside the container - you can supply env variables (by a file or otherwise) to the container from the host when you run it. Is it possible to make the .env file optional?

If not, we could hack it by creating the empty `.env` file within the Dockerfile itself, so it's not in the repo.

**Comment by Nopzen** [comment]
The issue is i would like to keep information secret within the .env file, eg api keys, and and ports and so forth to our database so people can't spam access them easily by reading them on the dockerfile :)

**Comment by Braedon** [comment]
They wouldn't be in the Dockerfile - they would be specified as part of the `docker run` command when you run a container.

e.g. `sudo docker run -d --name journalist -p 8080:8080 -e ELASTIC_HOST=<some server> journalist`

The Dockerfile only has generic defaults, where applicable (e.g. always bind to port 8080 inside the docker container - external port determined by port mapping).

You can also provide an env file to docker run.

e.g. `sudo docker run -d --name journalist -p 8080:8080 --env-file .env journalist`

**Pull request merged**